test/models/user_test.rb

```diff
@@ -95,4 +95,24 @@ class UserTest < ActiveSupport::TestCase
 
     assert_equal "Roger Federer", user.full_name
   end
+
+  test "it knows if it is a borrower" do
+    tenant = create(:tenant)
+    tenant.user << create(:user)
+    user = tenant.user.first
+
+    result = user.borrower?
+
+    assert result
+    refute user.lender?
+  end
+
+  test "it knows if it is a lender" do
+    user = create(:user)
+
+    result = user.lender?
```

Comment:
I kind of agree with Emily here.  In a test, the middle step is the "exercise".  Since we don't really exercise anything in these tests, there shouldn't be  a middle section of the test.